lessons/12-css-pseudo-classes/index.html

```diff
@@ -0,0 +1,41 @@
+<html>
+    <head>
+       <title>Pseudo Classes</title>
+       <style>
+           div li:nth-child(2) :hover {
```

Comment:
Oops! There shouldn't be a space between `:hover` and `:nth-child(2)`

`div li:nth-child(2):hover`